lib/PrintAsClang/DeclAndTypePrinter.cpp

```diff
@@ -1110,6 +1110,12 @@ class DeclAndTypePrinter::Implementation
 
   void visitFuncDecl(FuncDecl *FD) {
     if (outputLang == OutputLanguageMode::Cxx) {
+      // Don't expose async functions or @_alwaysEmitIntoClient functions
+      // because they're currently unsupported
+      if (FD->hasAsync() || FD->getAttrs().hasAttribute<AlwaysEmitIntoClientAttr>()) {
```

Comment:
One final NIT: Please format this line to follow the 80 columns style limit. For example you can run something like this:

```
git diff -U0 --no-prefix HEAD~1 HEAD -- lib/ include/ | ~/llvm-project/clang/tools/clang-format/clang-format-diff.py -style LLVM -i -binary $(xcrun --find clang-format)
```

Reply:
Somehow I didn't get any output from this command, and `git status` shows there's no change.

Here's my version of this command:

```bash
git diff -U0 --no-prefix HEAD~1 HEAD -- lib/ include/ | ../llvm-project/clang/tools/clang-format/clang-format-diff.py -style LLVM -i -binary $(xcrun --find clang-format)
```

And it's run inside `swift-project/swift`.

Reply:
I managed to manually invoke clang-format and I'm going to push the commit.

Reply:
I think I know why it doesn't work. `HEAD~1` didn't change DeclAndTypePrinter.cpp so there's no output from it.